.travis.yml

```diff
@@ -85,7 +85,7 @@ before_script:
 
 # Run PHPs run-tests.php
 script:
-    - ./travis/test.sh -d opcache.jit_buffer_size=16M -d opcache.jit=tracing
+    - travis_wait 60 ./travis/test.sh -d opcache.jit_buffer_size=64M -d opcache.jit=tracing
```

Comment:
`travis_wait` wasn't necessary because we were still printing skipped tests for PHP 8.1 (and PHP 8.2 I think?)

Reply:
I'd like to use the same parameters everywhere, to avoid hitting any limit and not compiling any trace, in case the unit tests were to include some bigger integration tests in the future.